### PR TITLE
TASK: Add integration test debugging hints to readme

### PR DESCRIPTION
**Pull request by Sebobo**
Some steps on how to find out why a test is failing.

### CODE REVIEW

**Comment by Sebobo** [comment]
I created this as draft. I'm sure there is more to add.

File: README.md [hunk 1]
```diff
@@ -172,6 +172,12 @@ To setup end-to-end tests locally you have got to do the same things described i
 
 For executing the end to end tests on a Mac with catalina or higher you need to permit screen recording. Open 'System Preferences > Security & Privacy > Privacy > Screen Recording' and check 'TestCafe Browser Tools' in the application list.
 
+##### Debugging integration tests
+
+* View the recording via saucelabs. You can find the url in the beginning of the test output.
+* Observe exceptions and logs in build artifacts.
```
Comment (dimaip) [inline README.md]:
```suggestion
* Observe Flow exceptions and logs in build artifacts.
```

File: README.md [hunk 2]
```diff
@@ -172,6 +172,12 @@ To setup end-to-end tests locally you have got to do the same things described i
...
+* You can trigger a SSH enabled build via the circle ci interface and then login.
```
Comment (dimaip) [inline README.md]:
```suggestion
* You can trigger a SSH enabled build via the CircleCI interface and then login.
```

File: README.md [hunk 3]
```diff
@@ -172,6 +172,12 @@ To setup end-to-end tests locally you have got to do the same things described i
 
 For executing the end to end tests on a Mac with catalina or higher you need to permit screen recording. Open 'System Preferences > Security & Privacy > Privacy > Screen Recording' and check 'TestCafe Browser Tools' in the application list.
 
+##### Debugging integration tests
+
+* View the recording via saucelabs. You can find the url in the beginning of the test output.
```
Comment (dimaip) [inline README.md]:
```suggestion
* View the recording via Sauce Labs. You can find the url in the beginning of the test output.
```

**Pull request merged**